README.md

```diff
@@ -0,0 +1,161 @@
+Triage
+======
+
+Risk modeling and prediction for public policy
+
+[![image](https://travis-ci.org/dssg/triage.svg?branch=master)](https://travis-ci.org/dssg/triage)
+[![image](https://codecov.io/gh/dssg/triage/branch/master/graph/badge.svg)](https://codecov.io/gh/dssg/triage)
+[![image](https://codeclimate.com/github/dssg/triage.png)](https://codeclimate.com/github/dssg/triage)
+
+Predictive analytics projects require the coordination of many different
+tasks, such as feature generation, classifier training, evaluation, and
+list generation. These tasks are complicated in their own right, but in
+addition have to be combined in different ways throughout the course of
+the project.
+
+Triage aims to provide interfaces to these different phases of a
+project, such as an `Experiment`. Each phase is defined by configuration
+specific to the needs of the project, and an arrangement of core data
+science components that work together to produce the output of that
+phase.
+
+`Experiment` (create features and models) -> `Audition` (pick the best models) -> `Postmodeling` (dive into best models)
+
+## Documentation Quick Links
```

Comment:
maybe just a quick sentence about what each of these is (e.g., "Start here if you want...")?